README.md

```diff
@@ -7,10 +7,17 @@
 [gemnasium-img-url]:https://gemnasium.com/rightscale/praxis.svg
 [gemnasium-url]:https://gemnasium.com/rightscale/praxis
 
-Praxis is a framework that takes a different approach to creating APIs, an approach that treats both designers and implementors as first class citizens and hold both their hands throughout the API building process. With Praxis you create an API by iterating through the design, review and implementation phases.
+Praxis is a framework for both _designing_ and _implementing_ APIs.
 
-Building APIs with Praxis will result in faster developer times, and result in very precise documentation that is apt for both human (web browsable) or machine consumption (JSON spec files). A very important feature of Praxis is that while the design and implementation phases are done independently, the resulting system will still ensure that they are always consistent and enforced at all times. This enforcement will automatically hold true throughout any number of iterations of the design-review-implementation phases. You can, once
-and for all, stop worrying about re-generating server code and keeping it in sync with any change to your code and or docs that you make.
+An important part of the framework is geared towards the _design_ aspect of building an API. This functionality empowers architects with tools to design every last aspect of their API, resulting in a complete, web-browsable documentation, which includes automatic generation of examples for resources, parameters, headers, etc... as well as requests and responses for the supported encodings. The design process is iterative, and flows from defining new resources, parameters, etc...to reviewing the resulting docs (usually with some of the potential clients of the API), back to updating the design based on feedback, or expanding it with more resources. The design language (i.e. DSL) of Praxis follows a clean 'ruby-type-syntax' and its final outcome is to generates an output that is both a set schema documents as well as a web-based API browser (driven by those schemas). You can use Praxis to design APIs that can be implemented in any language.
```

Comment:
space after etc...
set *of* schema documents
The last sentence might be better omitted, since we don't have any practical support for that.